source/_patterns/00-atoms/components/definition-list.mustache

```diff
@@ -1,8 +1,14 @@
-<dl class="definition-list{{#variant}} definition-list--{{variant}}{{/variant}}">
-  {{#items}}
-    <dt>{{{term}}}</dt>{{!
-    }}{{#descriptors}}{{!
-      }}<dd>{{{.}}}</dd>{{!
-    }}{{/descriptors}}
-  {{/items}}
-</dl>
+{{#isCollapsible}}
+  <div class="definition-list__wrapper">
```

Comment:
Do we need to introduce a wrapper of could be achieve the same behaviour with a class being added to the element with js. If you introduce the wrapper if property `isCollapsible` is true then you need to wrap a lot of styles in `.js` but this could be avoided by introducing a class with js.